platform/mac/mac.gradle

```diff
@@ -49,27 +42,19 @@ tasks.register('exportMacX64', Copy.class) {
         // chmod needed on executable as mkisofs does not +x on Nix systems
         if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
             exec { commandLine('chmod', '+x', exportPathX64 + '/MacOS/mcreator') }
+            exec { commandLine('chmod', '+x', 'platform/mac/disk/create-dmg.sh') }
         }
 
         // Create x64 DMG
-        ant.exec(executable: mkisofs_binary, failonerror: true) {

```

Comment:
Keep this one for Linux and Windows and decide which one to all based on OS type.

Also, don't make .sh script for this but define arguments inside ant.exec like we did for mkisofs

Reply:
> Keep this one for Linux and Windows and decide which one to all based on OS type.

What do you mean by this?

Reply:
I mean that Linux and Windows should still be able to generate DMG using mkiofs tool